contracts/axelar-interchain-token-service/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "axelar-interchain-token-service"
```

```suggestion
name = "interchain-token-service"
```
we don't really prefix ITS with axelar for other chains